deepchem/molnet/load_function/tests/test_molnet_loaders.py

```diff
@@ -0,0 +1,61 @@
+"""
+Tests for MolNet loader functions.
+"""
+
+import os
+import tempfile
+import shutil
+import numpy as np
+import deepchem as dc
+from deepchem.molnet import load_bandgap, load_perovskite
+
+# TODO: add unit tests for other dataset loaders that comply with
+# MolNet loader contribution template
+
+
+def test_material_dataset_loaders():
+  current_dir = os.path.dirname(os.path.abspath(__file__))
+  tasks, datasets, transformers = load_bandgap(
+      reload=False,
+      data_dir=current_dir,
```

Comment:
Ah this is clever! I really like this approach of pointing the loader to a dummy dataset that's of small size. Perhaps we could do the same for the doctests?

Reply:
Yeah this works pretty well! The tests only take a few seconds and the gzipped dummy datasets take hardly any space. Once these tests look good, I'm going to update the MolNet template with instructions on writing unit tests for new loaders.